benchmarks/asv_torch.conf.json

```diff
@@ -0,0 +1,154 @@
+{
+    // The version of the config file format.  Do not change, unless
+    // you know what you are doing.
+    "version": 1,
+
+    // The name of the project being benchmarked
+    "project": "kymatio",
+
+    // The project's homepage
+    "project_url": "https://kymat.io/",
+
+    // The URL or local path of the source code repository for the
+    // project being benchmarked
+    "repo": "..",
+
+    // Customizable commands for building, installing, and
+    // uninstalling the project. See asv.conf.json documentation.
+    //
+    //"install_command": [
+     //       "in-dir={env_dir} pyt",
+    //],
+    //     "in-dir={env_dir} python -mpip install {wheel_file}"],
+    // "uninstall_command": [
+    //     "return-code=any python -mpip uninstall -y {project}"],
+    //"build_command": [
+    //     "in-dir={env_dir} python setup.py install",],
+    //     "PIP_NO_BUILD_ISOLATION=false python -mpip wheel --no-deps --no-index -w {build_cache_dir} {build_dir}"
+    // ],
+
+    // List of branches to benchmark. If not provided, defaults to "master"
+    // (for git) or "default" (for mercurial).
+    //"branches": ["HEAD"], // for git
```

Comment:
`master` no (like above)?

Reply:
it should be commented out everywhere AFAIU